docs/usage.rst

```diff
@@ -460,3 +460,15 @@ Here is the same code using a context manager:
             assert os.path.exists(real_temp_file.name)
         assert not os.path.exists(real_temp_file.name)
         assert os.path.exists(fake_temp_file.name)
+
+Troubleshooting
+---------------
+
+OS temporary directories
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+As ``pyfakefs`` does not fake the ``tempfile`` module this means that a
+temporary directory is required to ensure ``tempfile`` works correctly.
+This means that any newly created fake file system will always have
+either a directory named ``/tmp`` when running on Linux or Unix systems
+or ``/var`` when running on MacOs.
```

Comment:
This looks good! I just would like to have two minor additions:
- mention that the temp directory that will be created is the one returned by `tempfile.gettempdir()` under each system (as generally there may be several temp directories) 
- add the Windows location of the temp dir (by default `C:\Users\<user>\AppData\Local\Temp`), and maybe add that under MacOS this temp dir is usually something like `/var/folders/<some cryptic ID>/T`

Reply:
Ah yes, I knew I was forgetting something: Windows! All good, I'll update accordingly.

Reply:
How's that?